src/machine/machine_esp32.go

```diff
@@ -363,38 +363,16 @@ func (spi SPI) Configure(config SPIConfig) error {
 	// (mode), among others.
 	var ctrl2Reg, userReg, pinReg uint32
 	// For mode configuration, see table 29 in the reference manual (page 128).
```

Comment:
I think I was referring to an older version of the TRM. Please see table 7-3 on page 125 of https://www.espressif.com/sites/default/files/documentation/esp32_technical_reference_manual_en.pdf.

Reply:
Updated comments.